lineage/figures/figure12.py

```diff
@@ -20,15 +20,6 @@
 tHMM_solver = tHMM(X=data[0], num_states=1)
 tHMM_solver.fit()
 
-constant_shape = [int(tHMM_solver.estimate.E[0].params[2]), int(tHMM_solver.estimate.E[0].params[4])]
```

Comment:
This file isn't yet updated to run `atonce`.

Reply:
Thank you, I will fix this in my branch.